packages/main/src/plugin/recommendations/recommendations-registry.ts

```diff
@@ -145,6 +154,12 @@ export class RecommendationsRegistry {
           default: false,
           hidden: false,
         },
+        [RecommendationsSettings.SectionName + '.' + RecommendationsSettings.IgnoreBannerRecommendations]: {
```

Comment:
nit: I think we have some rules encouraging to use string literals rather than concatenation

Reply:
Updated